.travis.yml

```diff
@@ -3,20 +3,18 @@ language: python
 sudo: false
 env:
   global:
-    - PYTHON_VERSION=2.7
+    - PYTHON_VERSION=3.5
```

Comment:
?

Reply:
Long story short: there is a bug in conda that the Qiime2 guys reported and this is the best way to install everything. See that these changes are in master and are passing.